docs/data/sdks/browser-2/index.md

```diff
@@ -144,11 +144,11 @@ envInstance.init(API_KEY_ENV, {
 
 Starting version 1.9.1, Browser SDK now tracks default events. Browser SDK can be configured to track the following events automatically:
 
-* Attribution
-* Page views
-* Sessions
-* Form interactions
-* File downloads
+- Attribution
+- Page views
+- Sessions
+- Form interactions
+- File downloads
```

Comment:
Thanks for using markdownlint and fix "MD004/ul-style: Unordered list style [Expected: dash; Actual: asterisk]"